### PR TITLE
Added some documentation

### PR DESCRIPTION
**Pull request by hugolundin**
This really is not finished, but we decided that I should send a pull request because I will not have more time to work on it.

### CODE REVIEW

File: web/docs/source/GetStartedGuide.rst [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    cd chronos/web
+    python3 -m venv venv
+    . venv/bin/activate
+    pip install -r requirements.txt
+
+If needed, also install docker-compose:
+
+.. code-block:: bash
+
+    pip install docker-compose
```
Comment (Greenheart) [inline web/docs/source/GetStartedGuide.rst:26]: Is this the recomended way to install docker-compose on all platforms? Or should Linux-users still install from the docker website?

Comment (Limpan) [inline web/docs/source/GetStartedGuide.rst:26]: https://docs.docker.com/compose/install/#alternative-install-options doesn't elaborate that much but recommends against a system-wide install with pip.

https://docs.docker.com/compose/completion/ might also be of interest.

**Comment by Limpan** [comment]
Maybe we should put this in a branch of it's own? At least if we consider it a work in progress?

@hugolundin How complete would you consider it to be?

**Comment by hugolundin** [comment]
I don't really know how complete I consider it to be really. But it needs some work to be understandable for new developers (I think?)

**Comment by Limpan** [comment]
After looking through it a bit more thoroughly I think it would be best to merge it as it is. It's definitely good enough to get going and it will be easiest to improve if it's in the develop branch.

**Comment by Greenheart** [comment]
@teknik-eksjo/te13 Yay! This project isn't dead after all! :100: :100:

**Pull request merged**